tests/examples/test_scripts.py

```diff
@@ -94,6 +94,11 @@ def run_test(filepath):
             "translation.py",
             marks=pytest.mark.skipif(not _TEXT_AVAILABLE, reason="text libraries aren't installed")
         ),
+        # pytest.param(
```

Comment:
Could you add some tests ?

Reply:
By default, it runs fast_dev_run=True. I guess couple of tests to get the coverage up.
Plus, need to make sure the imports don t break if your lib is missing